packages/core/src/ui/UIGroup.ts

```
@@ -0,0 +1,208 @@
+        passDownFlags |= GroupModifyFlags.RaycastEnable;
+      }
+    }
+    if (!!flags) {
+      this._disorderedElements.forEach(
+        (element: IUIElement) => {
+          element._onGroupModify(passDownFlags);
+        },
+        () => {}
+      );
+    }
+    if (!!passDownFlags) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)